### ISSUE TITLE
unable to configure jwt after updating to dry-configurable 0.13.0

### ISSUE THREAD

**Issue by sideshowbandana** [waiting-for-dev/warden-jwt_auth#36]
## Expected behavior
The following should load properly:
```
Devise.setup do |config|
  config.jwt do |jwt|
    jwt.secret = ENV['DEVISE_JWT_SECRET_KEY']
    jwt.dispatch_requests = [
      ['POST', %r{^/login$}]
    ]
    jwt.revocation_requests = [
      ['DELETE', %r{^/logout$}]
    ]
    jwt.expiration_time = ENV.fetch("JWT_EXPIRATION_TIME", 1.year).to_i
  end
# ...
end
```

## Actual behavior
```
Failure/Error: require File.expand_path('../../config/environment', __FILE__)

NoMethodError:
  undefined method `each_with_object' for nil:NilClass
# /.../.rvm/gems/ruby-2.7.3/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:23:in `symbolize_keys'
# /.../.rvm/gems/ruby-2.7.3/gems/warden-jwt_auth-0.5.0/lib/warden/jwt_auth.rb:62:in `block in <module:JWTAuth>'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/setting.rb:147:in `evaluate'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/setting.rb:79:in `value'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `block in values'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/settings.rb:49:in `each'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/settings.rb:49:in `each'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `map'
# /.../.rvm/gems/ruby-2.7.3/gems/dry-configurable-0.13.0/lib/dry/configurable/config.rb:77:in `values'
# /.../.rvm/gems/ruby-2.7.3/gems/devise-jwt-0.8.1/lib/devise/jwt.rb:20:in `jwt'
# ./config/initializers/devise.rb:325:in `block in <main>'

```
## Steps to Reproduce the Problem

1. Update the the `dry-configurable` dependency to `0.13.0`

## Debugging information

Provide following information. Please, format pasted output as code. Feel free to remove the secret key value.

- Version of `warden-jwt_auth` in use
```
0.5.0
```
- Output of `Warden::JWTAuth.config`
```
N/A Unable to load
```

**Comment by xiaoboa** [comment]
duplicate issue of https://github.com/waiting-for-dev/warden-jwt_auth/issues/34

**Comment by waiting-for-dev** [comment]
Released on v0.6.0
